api/serializers.py

```
@@ -0,0 +1,36 @@
+from rest_framework import serializers
+from .models import (
+    Word, Group, WordGroup, StudyActivity, StudySession, WordReviewItem,
```

Ruff (0.8.2)

`.models.Group` imported but unused

Remove unused import

(F401)